src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -100,6 +100,8 @@ export type HostConfig<T, P, I, TI, PI, C, CX, PL> = {
   resetAfterCommit() : void,
 
   useSyncScheduling ?: boolean,
+
+  mockComponent ?: Function,
```

Comment:
I was guided towards making the `HostConfig` shape strict in a prior change. May be applicable here as well.

See https://github.com/facebook/react/pull/8628#discussion_r93817157